modules/serverless-installing-knative-serving.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+[id="installing-knative-serving_{context}"]
+= Installing Knative Serving
+
+You need to create a `KnativeServing` object to install Knative Serving using the Openshift Serverless Operator.
```

Comment:
There are a lot of instances throughout the PR of `Openshift` (lowercase `s`). They should all be `OpenShift`.